### DISCUSSION TITLE
对齐function call后，如何评估？指标是什么？

### DISCUSSION THREAD

**Discussion by DryPilgrim** [zai-org/ChatGLM3#254] [General]
1. 如题，glm3对齐function call后，如何评估的呀？指标是什么？
2. 有评估脚本吗？
3. 如果不方便说指标设计和脚本，可以说一下评估思想吗？

非常感谢您的回答：）

**Comment by zRzRzRzRzRzRzR** [comment]
目前没有相关的关于glm3-6b的Function Call评测指标，如果有，我们会第一时间更新第三方公布的Function Call 评价
目前，我们还没有公开任何相关的官方指标数据和评测标准，如果有，我们也会第一时间更新
感谢你的关注
